components/http_foundation.rst

```diff
@@ -163,20 +163,25 @@ exist::
     // returns 'baz'
 
 When PHP imports the request query, it handles request parameters like
-``foo[bar]=baz`` in a special way as it creates an array. So you can get the
-``foo`` parameter and you will get back an array with a ``bar`` element::
+``foo[bar]=baz`` in a special way as it creates an array. The ``get()`` method
+doesn't support returning arrays, so you need to use the following code::
 
     // the query string is '?foo[bar]=baz'
 
-    $request->query->get('foo');
+    $request->query->all()['foo'];
     // returns ['bar' => 'baz']
 
     $request->query->get('foo[bar]');
     // returns null
 
-    $request->query->get('foo')['bar'];
+    $request->query->all()['foo']['bar'];
     // returns 'baz'
 
+.. versionadded:: 5.1
+
+    The feature that removed array support in ``get()`` was introduced in
```

Comment:
> Support for retrieving array values in ``get()`` was removed in Symfony 5.1.

Would make the sentence a bit more readable. Because adding a feature to remove something reads ... weird 😄 
(maybe use "deprecated" instead of "removed"? Also the directive should probably be the "deprecated" one)

Reply:
You are completely right! Thanks for telling me. I've updated this.